cfme/infrastructure/virtual_machines.py

```diff
@@ -872,3 +1094,12 @@ class Timelines(CFMENavigateStep):
 
     def step(self):
         mon_btn('Timelines')
+
+
+@navigator.register(Vm, 'Reconfigure')
+class VmReconfigure(CFMENavigateStep):
+    VIEW = InfraVmReconfigureView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        cfg_btn('Reconfigure this VM')
```

Comment:
I would need a VMDetails view for that which we don't currently have and I'd rather avoid doing that in this PR.